app/docs/0.10.x/configuration.md

```diff
@@ -12,6 +12,7 @@ title: Configuration Reference
 - [Custom Nginx configuration & embedding Kong](#custom-nginx-configuration-embedding-kong)
   - [Custom Nginx configuration](#custom-nginx-configuration)
   - [Embedding Kong](#embedding-kong)
+  - [Serving API and Website](#serving-api-and-website)
```

Comment:
We should use a slightly more descriptive title here, like:

"Serving both a website and your APIs from Kong"